defaults/main.yml

```diff
@@ -1,12 +1,13 @@
 ---
 i2_manage_repository: true
 i2_apt_key: "https://packages.icinga.com/icinga.key"
-i2_apt_url: "deb http://packages.icinga.com/{{ ansible_distribution|lower }} icinga-{{ ansible_distribution_release }} main"
+i2_apt_url: "deb http://packages.icinga.com/{{ ansible_lsb.id|lower }} icinga-{{ ansible_distribution_release }} main"
```

Comment:
The reason I changed this is because this works for e.g. the raspbian release of icinga2 whereas ansible_distribution would always return `Debian`.